sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -69,10 +69,10 @@ def content_type(self):
 
     @property
     def id(self):
-        # type: () -> str
+        # type: () -> Optional[str]
         """The secret's id
 
-        :rtype: str
+        :rtype: optional[str]
```

Comment:
nit: I think this is a bit clearer in generated documentation (this applies to all optional return types)
```suggestion
        :rtype: str or None
```

Reply:
Make sense, I'll update them to be this style.